styles.css

```diff
@@ -230,14 +230,26 @@ figure:hover figcaption {
   justify-content: space-around;
 }
 
-.information {
+.about-us .flex-column-container {
```

Comment:
tokiu atveju geriau pridėti papildomą klasę ant to paties elemento, o ne papildyti flex-container. nes flex-comtainer jau iš pavadinimo turi lavai aiškią paskirtį.